pages/common/conda-notices.md

```diff
@@ -0,0 +1,16 @@
+# conda notices
+
+> Retrieve latest channel notifications.
+> More information: <https://docs.conda.io/projects/conda/en/latest/commands/notices.html#>.
+
+- Show notices for the default channel and all `.condarc` channels:
+
+`conda notices`
+
+- Include a specific channel:
+
+`conda notices {{[-c|--channel]}} {{channel_name}}`
+
+- Ignore default and `.condarc` channels. Requires -c flag:
```

Comment:
```suggestion
- Ignore default and `.condarc` channels:
```
How important is this information?

Reply:
We're not a manual though. We provide fully functional commands which is why adding that something is mandatory isn't needed. There are plenty of commands with mandatory extra bits but it isn't stated.